config/initializers/propshaft.rb

```diff
@@ -1,3 +1,7 @@
 # frozen_string_literal: true
 
+# SVG Icons
 Rails.application.config.assets.paths << Rails.root.join('app', 'javascript', 'images')
+
+# Tell propshaft where the material design icons are
+Rails.application.config.assets.paths << Rails.root.join('app', 'javascript', 'material-icons', '400-24px')
```

Comment:
I don't think the full path should be included this way. Either just `app/javascript/` or `app/javascript/material-icons` imho.

Reply:
One issue with `app/javascript` is that during asset compilation propshaft will copy/digest all the stuff in these paths into public/assets. That's probably harmless if it's a few svg images that we're using in the JS app but not the Rails app, but if it's the entirety of the JS, that's going to be undesirable both time/size wise.

Pushed a change which adjusts the path to just material-icons, and updates the helper to include the size/dir name.

Reply:
> One issue with `app/javascript` is that during asset compilation propshaft will copy/digest all the stuff in these paths into public/assets. That's probably harmless if it's a few svg images that we're using in the JS app but not the Rails app, but if it's the entirety of the JS, that's going to be undesirable both time/size wise.

Does this mean this behavior PR's copies used icons twice? All the icons once via propshaft, and all the icons used in the React app through webpack?